From eb8c454b66fa5f522c5d49c51d05bc0a3ea6dea3 Mon Sep 17 00:00:00 2001
From: Olaf Hering <olaf@aepfle.de>
Date: Sun, 28 Apr 2019 12:34:56 +0000
Subject: [PATCH] [cmake] remove usage of kodi-platform

The vfs.nfs binary addon does not use kodi-platform.
Remove existing references from CMakeLists.txt and debian/control.
Add dependency to libp8-platform-dev because libkodiplatform-dev
pulled that dependency in as a side effect. Fix that bug too by
adding an explicit dependency.

Signed-off-by: Olaf Hering <olaf@aepfle.de>

Downloaded from upstream PR 11
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 CMakeLists.txt | 4 +---
 debian/control | 2 +-
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 034b2f4..9153b3d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,10 +9,8 @@ enable_language(CXX)
 find_package(NFS REQUIRED)
 find_package(Kodi REQUIRED)
 find_package(p8-platform REQUIRED)
-find_package(kodiplatform REQUIRED)
 
 include_directories(${KODI_INCLUDE_DIR}/.. # Hack way with "/..", need bigger Kodi cmake rework to match right include ways
-                    ${kodiplatform_INCLUDE_DIRS}
                     ${p8-platform_INCLUDE_DIRS}
                     ${NFS_INCLUDE_DIRS})
 
@@ -22,7 +20,7 @@ set(NFS_HEADERS src/NFSConnection.h
 set(NFS_SOURCES src/NFSConnection.cpp
                 src/NFSFile.cpp)
 
-set(DEPLIBS ${NFS_LIBRARIES} ${p8-platform_LIBRARIES} ${kodiplatform_LIBRARIES})
+set(DEPLIBS ${NFS_LIBRARIES} ${p8-platform_LIBRARIES})
 
 add_definitions(-D_LARGEFILE64_SOURCE
                 -D_FILE_OFFSET_BITS=64
diff --git a/debian/control b/debian/control
index af8aa56..1f64ed0 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: kodi-vfs-nfs
 Priority: extra
 Maintainer: Arne Morten Kvarving <spiff@xbmc.org>
 Build-Depends: debhelper (>= 9.0.0), cmake, libnfs-dev,
-               libkodiplatform-dev (>= 17.0.0), kodi-addon-dev
+               libp8-platform-dev, kodi-addon-dev
 Standards-Version: 3.9.4
 Section: libs
 
